src/screens/MemberDetail/MemberDetail.tsx

```
@@ -57,207 +48,142 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
-                pageInfo: fetchMoreResult.user.user.tagsAssignedWith.pageInfo,
-                totalCount: fetchMoreResult.user.user.tagsAssignedWith.pageInfo,
-              },
-            },
-          },
-        };
-      },
-    });
+  // Function to handle the click on the edit icon
+  const handleEditIconClick = (e: React.MouseEvent): void => {
+    const input = e.target as HTMLInputElement;
+    if (input.files && input.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)